src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You have specified 'Roboto' as the font-family, but it's good practice to include a generic font family as a backup. For example: font-family: Roboto, Arial, sans-serif, 'sans-serif';